pkg/ccl/workloadccl/fixture.go

```diff
@@ -417,12 +417,34 @@ func importFixtureTable(
 	}
 	var rows, index, tableBytes int64
 	var discard driver.Value
-	err := sqlDB.QueryRow(buf.String(), params...).Scan(
-		&discard, &discard, &discard, &rows, &index, &tableBytes,
-	)
+	res, err := sqlDB.Query(buf.String(), params...)
```

Comment:
Would it be better to name the columns and avoid the branching below? Of course only if the column names never changed.

Reply:
(`SELECT foo, bar, baz FROM [ IMPORT TABLE ... ]`)

Reply:
heh, I tied that first actually -- doesn't work since the optimizer doesn't know about import/restore apparently.